rest/config.go

```diff
@@ -696,6 +696,15 @@ func (dbConfig *DbConfig) validateVersion(isEnterpriseEdition bool) error {
 		}
 	}
 
+	if validateOIDCConfig && dbConfig.OIDCConfig != nil {
+		for name, provider := range dbConfig.OIDCConfig.Providers {
+			_, _, err := provider.DiscoverConfig(context.TODO())
```

Comment:
Is there a better context to use here? `validateVersion` doesn't take in a context, and other functions in `config.go` also use `context.TODO()`.

Reply:
Grabbing the context off the handler from the HTTP request (`h.rq.Context()`) and passing it down into validate would probably be worth doing, given the amount of work being done and logged inside `DiscoverConfig`.